src/pages/RepoPage/CoverageOnboarding/IntroBlurb/IntroBlurb.tsx

```diff
@@ -4,12 +4,17 @@ import Icon from 'ui/Icon'
 const IntroBlurb = () => {
   return (
     <div
-      className="flex flex-col gap-2 overflow-auto border-l-4 border-ds-blue bg-ds-gray-primary p-4"
+      className="mb-4 flex flex-col gap-2 overflow-auto border-l-4 border-ds-blue bg-ds-gray-primary p-4"
```

Comment:
`l`: I'd be careful when using margin's especially in vertical axis you can read more about there [here](https://www.joshwcomeau.com/css/rules-of-margin-collapse/). It's alright to leave, but moving forward I would try your best to avoid using margin as much as possible cause it can be unpredictable.